public/scripts/char-data.js

```diff
@@ -0,0 +1,340 @@
+class WorldInfoEntry {
```

Comment:
I prefer to use JSDoc typedefs instead of classes.

Reply:
Found a tool
https://rafistrauss.github.io/jsdoc-generator/
Give me a minute, I'll try it.

Reply:
fixed in [`1d0e292` (#2209)](https://github.com/SillyTavern/SillyTavern/pull/2209/commits/1d0e29278b76e90b12dd5cab50f35ebf98b363cd)